app/src/main/java/de/westnordost/streetcomplete/osm/Things.kt

```diff
@@ -241,7 +241,6 @@ private val IS_THING_EXPRESSION by lazy {
         or attraction
         or boundary = marker
         or leisure = pitch and sport ~ chess|table_soccer|table_tennis|teqball
-        or man_made = street_cabinet and street_cabinet != postal_service
```

Comment:
Nooo :persevere: 
Why remove street cabinets from the Things overlay?

Reply:
because all street cabinets are added.

Reply:
Ah, I see now. Sorry for the confusion.